chapters/en/slides/module7/module7_05.md

```diff
@@ -0,0 +1,202 @@
+Find out more about JupyterLab extensions [here](https://jupyterlab.readthedocs.io/en/stable/user/extensions.html).
+
+Notes:
+
+JupyterLab is built to be modular and extensible by nature. When a feature does not already exist in JupyterLab, developers can create an **extension** to provide that feature for JupyterLab.
+
+Do you remember using the Git extension in JupyterLab in previous chapters? Well, as the name suggests, that was an extension that was added on to JupyterLab when you followed the installation guide in Assignment 0!
+
+As a matter of fact, JupyterLab itself is nothing but a collection of various extensions.
+
+---
+
+# Installing JupyterLab extensions
+
+<br>
+
+- JupyterLab extensions may be be either **source** or **prebuilt** extensions
+
+- Since extensions are created using **JavaScript**:
+
+  - Installing a **source** extension requires a rebuild of JupyterLab, which in turn needs **Node.js** to be installed in your conda environment. You can install Node.js by running `conda install nodejs`.
+
+  - **Prebuilt** extensions can be used right away
+
+<br>
+
+In general, there are three ways to install JupyterLab extensions:
+
+1. Using `conda install <extension_name>`
+
+2. The `jupyter labextension install <extension_name>` command
+
+3. JupyterLab's Extension Manager GUI
+
+<br>
+
+<p style="color:black; font-size:110%; background-color:floralwhite;">
+Always refer to an extension's documentation to find the recommended way of installation.
+</p>
+
+Notes:
+
+In case it is the recommended method, installing an extension using 
+
+`conda install <extension_name>`
+
+or
+
+`jupyter labextension install <extension_name>`
+
+is straight-forward. Simply run the above commands in the Conda environment where JupyterLab is installed. Be sure to follow the steps recommended by the extension's documentation.
+
+Here is the [link](https://github.com/jupyterlab/jupyterlab-git#install) to the installation steps recommended by the `jupyterlab-git` extension. This is an example of a conda installation.
+
+---
+
+# JupyterLab Extension Manager
+
+<img src="/module7/ext2.png" width="700"></img>
+
+Notes:
+
+JupyterLab provides a GUI for managing extensions.
+
+As shown in the screenshots, you can view a list of installed extensions by clicking on the puzzle-piece icon on the left-hand side bar of JupyterLab.
+
+Suppose that we're looking for a particular extension called "jupyterlab-drawio". If you type "draw" in the search box at the top and hit `Enter`, you'll see that this extension appears at the top. Now you just need to click "Install", and your desired extension will be installed.
+
+Note that, if the extension you're trying to install is **not prebuilt**, you will be prompted with a message box to rebuild JupyterLab. But no need to worry! Simply click the "Rebuild" button. Finally, you'll see a "Build Complete" message if everything is successful. It's safest to click "Save and Reload" so that you don't lose any unsaved changes.
```

Comment:
I'm not sure we can install prebuilt extensions from the GUI, because I remember getting asked for rebuilding JupyterLab whenever I do that. I change the sentence saying:

> Note that, if the extension you're trying to install is **not prebuilt**, ...

to

> Note that, since extensions that you can install using the GUI are **not prebuilt**, ...

Reply:
Yes exactly, great!